include/fw_info.rst

```diff
@@ -29,6 +29,9 @@ It includes the following information:
 * The single, monotonically increasing version number of the image.
 * The address through which to boot into the firmware (the vector table address).
   This address is not necessarily the start of the image.
+* A value that must be 0xFFFFFFFF.
+  If this is set to 0 (e.g. by the bootloader), the image can quickly be established as invalid.
```

Comment:
In practice it needs only be different from `0xFFFFFFFF`.

Reply:
If the code enforce that non `0xFFFFFFFF` implies invalid, then IMO we should state that in the docs, otherwise it could be considered a bug that `0xFFFFFFF0` is considered invalid.

Reply:
Updated docs